tools/commands-packages-query.js

```diff
@@ -0,0 +1,1159 @@
+  // - version: (mandatory) package version
+  // - summary: version summary/short description (from Package.describe)
+  // - publishedBy: username of the publisher
+  // - publishedOn: date of publication
+  // - description: long-form description (from the README.md)
+  // - git: git URL for this version.
+  // - local: true for a local version of a package.
+  // - directory: source directory of this package.
+  // - installed: true if the package exists in warehouse, and is therefore
+  //   available for use offline.
+  // - architectures: if self.showOSArchitectures is true, returns an
+  //   array of system architectures for which that package is available.
+  // - dependencies: an array of package dependencies, as objects with the
+  //   following keys:
+  //     - packageName: name of the dependency
+  //     - constraint: constraint for that dependency
+  //     - weak: true if this is a weak dependency.
+  _displayVersion: function (data) {
+    var self = this;
+    Console.info("Package:", data.name);
+    Console.info("Version:", data.version);
+    if (data.summary) {
+      Console.info("Summary:", data.summary);
+    }
+    if (data.publishedBy) {
+      var publisher = data.publishedBy;
+      var pubDate = utils.longformDate(data.publishedOn);
+      Console.info("Published by", publisher, "on", pubDate);
+    }
+    if (data.git) {
+      Console.info("Git:", Console.url(data.git));
+    }
+    if (data.directory) {
+      Console.info("Directory:", Console.path(data.directory));
+    }
+    Console.info();
+
+    // If we don't have a long-form description, print the summary. (If we don't
+    // have a summary, print nothing).
+    if (data.description || data.summary) {
+      Console.info(data.description || data.summary);
+      Console.info();
+    }
+
+    // Print dependency information, if the package has any dependencies.
+    if (_.has(data, "dependencies") &&
+        ! _.isEmpty(data.dependencies)) {
+      Console.info("Depends on:");
+      _.each(data.dependencies, function (dep) {
+        var depString = dep.name;
+        if (dep.constraint && dep.constraint !== null) {
+          depString += "@" + dep.constraint;
+        }
+        if (dep.weak) {
+          depString += " (weak dependency)";
+        }
+        Console.info(
+          Console.noWrap(depString),
+          Console.options({ indent: 2 }));
+      });
+    }
+
+    // Sometimes, there is a server package and a local package with the same
+    // version. In this case, we prefer the local package. Explain our choice to
+    // the user.
+    if (data.local &&
+        catalog.official.getVersion(data.name, data.version)) {
+      Console.info();
+      Console.info(
+        "The same version of this package also exists on the package server.",
+        "To view its metadata, run",
+        Console.command("'meteor show " + data.name + "@" + data.version + "'"),
+        "from outside the project.");
```

Comment:
Yeah.  We thought about that when we were discussing the wording, but decided that anyone using those overrides would already know that. Certainly, mentioning PACKAGE_DIRS in this message seems like too much information and it would be nice to give some hint as to what to do next.